core/class-maxi-styles.php

```
@@ -172,8 +182,11 @@ public function enqueue_styles()
                 $template_parts_meta = [];
 
                 if ($template_parts && !empty($template_parts)) {
+                    // Bulk fetch template parts meta
+                    $template_parts_meta_bulk = [];
```

PHPMD (2.15.0)

Avoid unused local variables such as '$template_parts_meta_bulk'. (undefined)

(UnusedLocalVariable)